apps/drawio/24.7.17/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   drawio:
-    image: jgraph/drawio:24.7.17
+    image: jgraph/drawio:26.0.0
     container_name: ${CONTAINER_NAME}
     restart: unless-stopped
     networks:
```

Comment:
The provided code does not contain any evident irregularities, issues, or significant optimizations beyond updating the version of `jgraph/drawio` from 24.7.17 to 26.0.0.